lib/praxis/docs/open_api/schema_object.rb

```diff
@@ -8,12 +8,19 @@ class SchemaObject
         attr_reader :type
 
         def initialize(info:)
-          # info could be an attribute ... or a type?
-          @type =  info.is_a?(Attributor::Attribute) ? info.type : info
```

Comment:
Any removal of a use of the ternary operator gets a heavy +1 from me.

Reply:
👍